packages/replace/README.md

```diff
@@ -65,6 +65,13 @@ Default: `['\b', '\b']`
 
 Specifies the boundaries around which strings will be replaced. By default, delimiters are [word boundaries](https://www.regular-expressions.info/wordboundaries.html). See [Word Boundaries](#word-boundaries) below for more information.
 
+### `preventAssignment`
+
+Type: `Boolean`<br>
+Default: `false`
+
+Prevents replacing strings where they are followed by a single equals sign (e.g. `process.env.DEBUG = false`).
```

Comment:
let's insert the same example from the PR here for quick reference for users. when we flip the bit on this feature, a good example in the README will be important.